nopolinfo #0
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.IngestTest', failingMethods=[com.cloudera.oryx.app.serving.als.IngestTest#testFormIngest, com.cloudera.oryx.app.serving.als.IngestTest#testGZippedIngest, com.cloudera.oryx.app.serving.als.IngestTest#testDeflateIngest, com.cloudera.oryx.app.serving.als.IngestTest#testSimpleIngest, com.cloudera.oryx.app.serving.als.IngestTest#testZippedFormIngest, com.cloudera.oryx.app.serving.als.IngestTest#testGzippedFormIngest], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}], nbFailures=6, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:05 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@768ccdc5', projectTests=[com.cloudera.oryx.app.serving.als.IngestTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #1
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.PreferenceTest', failingMethods=[com.cloudera.oryx.app.serving.als.PreferenceTest#testPostJson, com.cloudera.oryx.app.serving.als.PreferenceTest#testPostWithEmptyItemValue, com.cloudera.oryx.app.serving.als.PreferenceTest#testPostWithBadItemValue, com.cloudera.oryx.app.serving.als.PreferenceTest#testPostCSV, com.cloudera.oryx.app.serving.als.PreferenceTest#testDelete], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<400> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}], nbFailures=5, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:11 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@7139bd31', projectTests=[com.cloudera.oryx.app.serving.als.PreferenceTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #2
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.AddTest', failingMethods=[com.cloudera.oryx.app.serving.kmeans.AddTest#testSimpleAdd, com.cloudera.oryx.app.serving.kmeans.AddTest#testFormAdd, com.cloudera.oryx.app.serving.kmeans.AddTest#testURIAdd], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}], nbFailures=3, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:17 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@3178219a', projectTests=[com.cloudera.oryx.app.serving.kmeans.AddTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #3
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.TrainTest', failingMethods=[com.cloudera.oryx.app.serving.rdf.TrainTest#testFormTrain, com.cloudera.oryx.app.serving.rdf.TrainTest#testSimpleTrain, com.cloudera.oryx.app.serving.rdf.TrainTest#testURITrain], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<204> but was:<500>', isError=false}], nbFailures=3, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:24 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@632b305d', projectTests=[com.cloudera.oryx.app.serving.rdf.TrainTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #4
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.AssignTest', failingMethods=[com.cloudera.oryx.app.serving.kmeans.AssignTest#testFormAssign, com.cloudera.oryx.app.serving.kmeans.AssignTest#testAssignPost], erroringMethods=[com.cloudera.oryx.app.serving.kmeans.AssignTest#testAssign2, com.cloudera.oryx.app.serving.kmeans.AssignTest#testAssign], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[4
4
]> but was:<[<html><head><title>Grizzly 2.3.28</title><style><!--div.header {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#003300;font-size:22px;-moz-border-radius-topleft: 10px;border-top-left-radius: 10px;-moz-border-radius-topright: 10px;border-top-right-radius: 10px;padding-left: 5px}div.body {font-family:Tahoma,Arial,sans-serif;color:black;background-color:#FFFFCC;font-size:16px;padding-top:10px;padding-bottom:10px;padding-left:10px}div.footer {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#666633;font-size:14px;-moz-border-radius-bottomleft: 10px;border-bottom-left-radius: 10px;-moz-border-radius-bottomright: 10px;border-bottom-right-radius: 10px;padding-left: 5px}BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;}B {font-family:Tahoma,Arial,sans-serif;color:black;}A {color : black;}HR {color : #999966;}--></style> </head><body><div class="header">Internal Error</div><div class="body"><b>com.google.common.collect.Sets$SetView.iterator()Lcom/google/common/collect/UnmodifiableIterator;</b><pre>     1: org.reflections.Reflections.expandSuperTypes(Reflections.java:380)
     2: org.reflections.Reflections.<init>(Reflections.java:126)
     3: org.reflections.Reflections.<init>(Reflections.java:168)
     4: org.reflections.Reflections.<init>(Reflections.java:141)
     5: com.cloudera.oryx.lambda.serving.OryxApplication.doGetClasses(OryxApplication.java:69)
     6: com.cloudera.oryx.lambda.serving.OryxApplication.getClasses(OryxApplication.java:57)
     7: org.glassfish.jersey.server.ResourceConfig$RuntimeConfig$3.run(ResourceConfig.java:1234)
     8: org.glassfish.jersey.internal.Errors$2.call(Errors.java:289)
     9: org.glassfish.jersey.internal.Errors$2.call(Errors.java:286)
    10: org.glassfish.jersey.internal.Errors.process(Errors.java:315)
        ... 18 more</pre>Please see the log for more detail.</div><div class="footer">Grizzly 2.3.28</div></body></html>]>', isError=false}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=2, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:29 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@9729a97', projectTests=[com.cloudera.oryx.app.serving.kmeans.AssignTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_ERRORING
----------



nopolinfo #5
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.AssignTest', failingMethods=[com.cloudera.oryx.app.serving.kmeans.AssignTest#testFormAssign, com.cloudera.oryx.app.serving.kmeans.AssignTest#testAssignPost], erroringMethods=[com.cloudera.oryx.app.serving.kmeans.AssignTest#testAssign2, com.cloudera.oryx.app.serving.kmeans.AssignTest#testAssign], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[4
4
]> but was:<[<html><head><title>Grizzly 2.3.28</title><style><!--div.header {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#003300;font-size:22px;-moz-border-radius-topleft: 10px;border-top-left-radius: 10px;-moz-border-radius-topright: 10px;border-top-right-radius: 10px;padding-left: 5px}div.body {font-family:Tahoma,Arial,sans-serif;color:black;background-color:#FFFFCC;font-size:16px;padding-top:10px;padding-bottom:10px;padding-left:10px}div.footer {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#666633;font-size:14px;-moz-border-radius-bottomleft: 10px;border-bottom-left-radius: 10px;-moz-border-radius-bottomright: 10px;border-bottom-right-radius: 10px;padding-left: 5px}BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;}B {font-family:Tahoma,Arial,sans-serif;color:black;}A {color : black;}HR {color : #999966;}--></style> </head><body><div class="header">Internal Error</div><div class="body"><b>com.google.common.collect.Sets$SetView.iterator()Lcom/google/common/collect/UnmodifiableIterator;</b><pre>     1: org.reflections.Reflections.expandSuperTypes(Reflections.java:380)
     2: org.reflections.Reflections.<init>(Reflections.java:126)
     3: org.reflections.Reflections.<init>(Reflections.java:168)
     4: org.reflections.Reflections.<init>(Reflections.java:141)
     5: com.cloudera.oryx.lambda.serving.OryxApplication.doGetClasses(OryxApplication.java:69)
     6: com.cloudera.oryx.lambda.serving.OryxApplication.getClasses(OryxApplication.java:57)
     7: org.glassfish.jersey.server.ResourceConfig$RuntimeConfig$3.run(ResourceConfig.java:1234)
     8: org.glassfish.jersey.internal.Errors$2.call(Errors.java:289)
     9: org.glassfish.jersey.internal.Errors$2.call(Errors.java:286)
    10: org.glassfish.jersey.internal.Errors.process(Errors.java:315)
        ... 18 more</pre>Please see the log for more detail.</div><div class="footer">Grizzly 2.3.28</div></body></html>]>', isError=false}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=2, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:34 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@43423d40', projectTests=[com.cloudera.oryx.app.serving.kmeans.AssignTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_FAILING
----------



nopolinfo #6
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.PredictTest', failingMethods=[com.cloudera.oryx.app.serving.rdf.PredictTest#testFormPredict, com.cloudera.oryx.app.serving.rdf.PredictTest#testPredictPost], erroringMethods=[com.cloudera.oryx.app.serving.rdf.PredictTest#testPredict, com.cloudera.oryx.app.serving.rdf.PredictTest#testPredict2], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[67.0
670.0
]> but was:<[<html><head><title>Grizzly 2.3.28</title><style><!--div.header {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#003300;font-size:22px;-moz-border-radius-topleft: 10px;border-top-left-radius: 10px;-moz-border-radius-topright: 10px;border-top-right-radius: 10px;padding-left: 5px}div.body {font-family:Tahoma,Arial,sans-serif;color:black;background-color:#FFFFCC;font-size:16px;padding-top:10px;padding-bottom:10px;padding-left:10px}div.footer {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#666633;font-size:14px;-moz-border-radius-bottomleft: 10px;border-bottom-left-radius: 10px;-moz-border-radius-bottomright: 10px;border-bottom-right-radius: 10px;padding-left: 5px}BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;}B {font-family:Tahoma,Arial,sans-serif;color:black;}A {color : black;}HR {color : #999966;}--></style> </head><body><div class="header">Internal Error</div><div class="body"><b>com.google.common.collect.Sets$SetView.iterator()Lcom/google/common/collect/UnmodifiableIterator;</b><pre>     1: org.reflections.Reflections.expandSuperTypes(Reflections.java:380)
     2: org.reflections.Reflections.<init>(Reflections.java:126)
     3: org.reflections.Reflections.<init>(Reflections.java:168)
     4: org.reflections.Reflections.<init>(Reflections.java:141)
     5: com.cloudera.oryx.lambda.serving.OryxApplication.doGetClasses(OryxApplication.java:69)
     6: com.cloudera.oryx.lambda.serving.OryxApplication.getClasses(OryxApplication.java:57)
     7: org.glassfish.jersey.server.ResourceConfig$RuntimeConfig$3.run(ResourceConfig.java:1234)
     8: org.glassfish.jersey.internal.Errors$2.call(Errors.java:289)
     9: org.glassfish.jersey.internal.Errors$2.call(Errors.java:286)
    10: org.glassfish.jersey.internal.Errors.process(Errors.java:315)
        ... 18 more</pre>Please see the log for more detail.</div><div class="footer">Grizzly 2.3.28</div></body></html>]>', isError=false}], nbFailures=2, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:39 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@7028b5cc', projectTests=[com.cloudera.oryx.app.serving.rdf.PredictTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_ERRORING
----------



nopolinfo #7
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.PredictTest', failingMethods=[com.cloudera.oryx.app.serving.rdf.PredictTest#testFormPredict, com.cloudera.oryx.app.serving.rdf.PredictTest#testPredictPost], erroringMethods=[com.cloudera.oryx.app.serving.rdf.PredictTest#testPredict, com.cloudera.oryx.app.serving.rdf.PredictTest#testPredict2], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[67.0
670.0
]> but was:<[<html><head><title>Grizzly 2.3.28</title><style><!--div.header {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#003300;font-size:22px;-moz-border-radius-topleft: 10px;border-top-left-radius: 10px;-moz-border-radius-topright: 10px;border-top-right-radius: 10px;padding-left: 5px}div.body {font-family:Tahoma,Arial,sans-serif;color:black;background-color:#FFFFCC;font-size:16px;padding-top:10px;padding-bottom:10px;padding-left:10px}div.footer {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#666633;font-size:14px;-moz-border-radius-bottomleft: 10px;border-bottom-left-radius: 10px;-moz-border-radius-bottomright: 10px;border-bottom-right-radius: 10px;padding-left: 5px}BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;}B {font-family:Tahoma,Arial,sans-serif;color:black;}A {color : black;}HR {color : #999966;}--></style> </head><body><div class="header">Internal Error</div><div class="body"><b>com.google.common.collect.Sets$SetView.iterator()Lcom/google/common/collect/UnmodifiableIterator;</b><pre>     1: org.reflections.Reflections.expandSuperTypes(Reflections.java:380)
     2: org.reflections.Reflections.<init>(Reflections.java:126)
     3: org.reflections.Reflections.<init>(Reflections.java:168)
     4: org.reflections.Reflections.<init>(Reflections.java:141)
     5: com.cloudera.oryx.lambda.serving.OryxApplication.doGetClasses(OryxApplication.java:69)
     6: com.cloudera.oryx.lambda.serving.OryxApplication.getClasses(OryxApplication.java:57)
     7: org.glassfish.jersey.server.ResourceConfig$RuntimeConfig$3.run(ResourceConfig.java:1234)
     8: org.glassfish.jersey.internal.Errors$2.call(Errors.java:289)
     9: org.glassfish.jersey.internal.Errors$2.call(Errors.java:286)
    10: org.glassfish.jersey.internal.Errors.process(Errors.java:315)
        ... 18 more</pre>Please see the log for more detail.</div><div class="footer">Grizzly 2.3.28</div></body></html>]>', isError=false}], nbFailures=2, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:46 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@55574295', projectTests=[com.cloudera.oryx.app.serving.rdf.PredictTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_FAILING
----------



nopolinfo #8
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.ReadyTest', failingMethods=[com.cloudera.oryx.app.serving.kmeans.ReadyTest#testHead, com.cloudera.oryx.app.serving.kmeans.ReadyTest#testGet], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}], nbFailures=2, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:48:58 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@6018d82c', projectTests=[com.cloudera.oryx.app.serving.kmeans.ReadyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #9
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.ReadOnlyTest', failingMethods=[com.cloudera.oryx.app.serving.als.ReadOnlyTest#testNoPreference, com.cloudera.oryx.app.serving.als.ReadOnlyTest#testNoIngest], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<403> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<403> but was:<500>', isError=false}], nbFailures=2, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:49:10 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@3b60be3', projectTests=[com.cloudera.oryx.app.serving.als.ReadOnlyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #10
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.ReadyTest', failingMethods=[com.cloudera.oryx.app.serving.rdf.ReadyTest#testHead, com.cloudera.oryx.app.serving.rdf.ReadyTest#testGet], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}], nbFailures=2, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:49:20 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@6ae55678', projectTests=[com.cloudera.oryx.app.serving.rdf.ReadyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #11
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.ReadyTest', failingMethods=[com.cloudera.oryx.app.serving.als.ReadyTest#testGet, com.cloudera.oryx.app.serving.als.ReadyTest#testHead], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}, FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<200> but was:<500>', isError=false}], nbFailures=2, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:49:29 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@2cc9a5a2', projectTests=[com.cloudera.oryx.app.serving.als.ReadyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #12
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.ConsoleTest', failingMethods=[com.cloudera.oryx.app.serving.als.ConsoleTest#testConsole], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<public> but was:<null>', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:49:35 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@6cf71064', projectTests=[com.cloudera.oryx.app.serving.als.ConsoleTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #13
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.ReadOnlyTest', failingMethods=[com.cloudera.oryx.app.serving.kmeans.ReadOnlyTest#testNoAdd], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<403> but was:<500>', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:49:40 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@7614beba', projectTests=[com.cloudera.oryx.app.serving.kmeans.ReadOnlyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #14
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.ConsoleTest', failingMethods=[com.cloudera.oryx.app.serving.rdf.ConsoleTest#testConsole], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<public> but was:<null>', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:49:47 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@35570a84', projectTests=[com.cloudera.oryx.app.serving.rdf.ConsoleTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #15
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.ReadOnlyTest', failingMethods=[com.cloudera.oryx.app.serving.rdf.ReadOnlyTest#testNoTrain], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<403> but was:<500>', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:01 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@114b81af', projectTests=[com.cloudera.oryx.app.serving.rdf.ReadOnlyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #16
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.ConsoleTest', failingMethods=[com.cloudera.oryx.app.serving.kmeans.ConsoleTest#testConsole], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='expected:<public> but was:<null>', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:09 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@1e64a724', projectTests=[com.cloudera.oryx.app.serving.kmeans.ConsoleTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #17
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.RecommendWithContextTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRecommendWithContextWithUnknown, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRecommendWithContext, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testBadHowMany, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRecommendCSV, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testTooMany, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testHowMany, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testBadArg, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testNoArg, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRescorer, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRecommendWithContextWithAllUnknown, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testBadOffset, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRecommendWithContextCSV, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testOffset, com.cloudera.oryx.app.serving.als.RecommendWithContextTest#testRecommend], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}], nbFailures=0, nbErrors=14}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:21 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@2f38b35e', projectTests=[com.cloudera.oryx.app.serving.als.RecommendWithContextTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #18
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testRecommendToAnonymousWithAllUnknown, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testOffset, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testHowMany, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testNoArg, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testBadOffset, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testRecommendToAnonymous, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testRecommendToAnonymousCSV, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testBadHowMany, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testRecommendToAnonymousWithUnknown, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testTooMany, com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest#testRescorer], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}], nbFailures=0, nbErrors=11}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:34 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@571010af', projectTests=[com.cloudera.oryx.app.serving.als.RecommendToAnonymousTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #19
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.RecommendTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.RecommendTest#testOffset, com.cloudera.oryx.app.serving.als.RecommendTest#testNoArg, com.cloudera.oryx.app.serving.als.RecommendTest#testHowMany, com.cloudera.oryx.app.serving.als.RecommendTest#testBadOffset, com.cloudera.oryx.app.serving.als.RecommendTest#testRecommendCSV, com.cloudera.oryx.app.serving.als.RecommendTest#testTooMany, com.cloudera.oryx.app.serving.als.RecommendTest#testConsiderKnownItems, com.cloudera.oryx.app.serving.als.RecommendTest#testBadHowMany, com.cloudera.oryx.app.serving.als.RecommendTest#testRecommend, com.cloudera.oryx.app.serving.als.RecommendTest#testRescorer], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}], nbFailures=0, nbErrors=10}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:42 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@4d574ec7', projectTests=[com.cloudera.oryx.app.serving.als.RecommendTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #20
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.RecommendToManyTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.RecommendToManyTest#testNoArg, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testBadOffset, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testTooMany, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testBadHowMany, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testHowMany, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testRescorer, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testRecommendToMany, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testOffset, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testRecommendToManyCSV, com.cloudera.oryx.app.serving.als.RecommendToManyTest#testConsiderKnownItems], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}], nbFailures=0, nbErrors=10}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:48 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@4cda59e6', projectTests=[com.cloudera.oryx.app.serving.als.RecommendToManyTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #21
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.SimilarityTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.SimilarityTest#testBadHowMany, com.cloudera.oryx.app.serving.als.SimilarityTest#testHowMany, com.cloudera.oryx.app.serving.als.SimilarityTest#testOffset, com.cloudera.oryx.app.serving.als.SimilarityTest#testBadOffset, com.cloudera.oryx.app.serving.als.SimilarityTest#testSimilarityCSV, com.cloudera.oryx.app.serving.als.SimilarityTest#testTooMany, com.cloudera.oryx.app.serving.als.SimilarityTest#testRescorer, com.cloudera.oryx.app.serving.als.SimilarityTest#testNoArg, com.cloudera.oryx.app.serving.als.SimilarityTest#testSimilarItems], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.BadRequestException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}], nbFailures=0, nbErrors=9}
status: NOPATCH
dateEnd: Wed Apr 19 00:50:56 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@754a38a0', projectTests=[com.cloudera.oryx.app.serving.als.SimilarityTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #22
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest#testEstimateForAnonymous, com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest#testEstimateForAnonymousWithUnknown, com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest#testUnknownUser, com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest#testBadRequest, com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest#testEstimateForAnonymousWithAllUnknown, com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest#testEstimateForAnonymousCSV], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}], nbFailures=0, nbErrors=6}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:03 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@4aaf547c', projectTests=[com.cloudera.oryx.app.serving.als.EstimateForAnonymousTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #23
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.MostSurprisingTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.MostSurprisingTest#testNoArg, com.cloudera.oryx.app.serving.als.MostSurprisingTest#testOffset, com.cloudera.oryx.app.serving.als.MostSurprisingTest#testHowMany, com.cloudera.oryx.app.serving.als.MostSurprisingTest#testMostSurprisingCSV, com.cloudera.oryx.app.serving.als.MostSurprisingTest#testMostSurprising], failures=[FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=5}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:10 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@6862aec0', projectTests=[com.cloudera.oryx.app.serving.als.MostSurprisingTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #24
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.BecauseTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.BecauseTest#testHowMany, com.cloudera.oryx.app.serving.als.BecauseTest#testBecause, com.cloudera.oryx.app.serving.als.BecauseTest#testBecauseCSV, com.cloudera.oryx.app.serving.als.BecauseTest#testNoArg, com.cloudera.oryx.app.serving.als.BecauseTest#testOffset], failures=[FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=5}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:18 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@2a1dabb6', projectTests=[com.cloudera.oryx.app.serving.als.BecauseTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #25
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.EstimateTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.EstimateTest#testZeroEstimate, com.cloudera.oryx.app.serving.als.EstimateTest#testEstimate, com.cloudera.oryx.app.serving.als.EstimateTest#testEstimateCSV, com.cloudera.oryx.app.serving.als.EstimateTest#testBadRequest], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=4}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:24 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@1b7a7ec8', projectTests=[com.cloudera.oryx.app.serving.als.EstimateTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #26
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.SimilarityToItemTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.SimilarityToItemTest#testNoArg, com.cloudera.oryx.app.serving.als.SimilarityToItemTest#testSimilarityToItemCSV, com.cloudera.oryx.app.serving.als.SimilarityToItemTest#testZeroSimilarityToItem, com.cloudera.oryx.app.serving.als.SimilarityToItemTest#testSimilarityToItem], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<javax.ws.rs.NotFoundException> but was<javax.ws.rs.InternalServerErrorException>', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=4}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:31 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@71dfcc99', projectTests=[com.cloudera.oryx.app.serving.als.SimilarityToItemTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #27
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.KnownItemsTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.KnownItemsTest#testEmptyItems, com.cloudera.oryx.app.serving.als.KnownItemsTest#testKnownItemsCSV, com.cloudera.oryx.app.serving.als.KnownItemsTest#testKnownItems], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=3}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:40 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@21dc2729', projectTests=[com.cloudera.oryx.app.serving.als.KnownItemsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #28
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.MostActiveUsersTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.MostActiveUsersTest#testRescorer, com.cloudera.oryx.app.serving.als.MostActiveUsersTest#testMostPopularCSV, com.cloudera.oryx.app.serving.als.MostActiveUsersTest#testMostPopular], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=3}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:48 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@1457dbd0', projectTests=[com.cloudera.oryx.app.serving.als.MostActiveUsersTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #29
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.MostPopularItemsTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.MostPopularItemsTest#testRescorer, com.cloudera.oryx.app.serving.als.MostPopularItemsTest#testMostPopular, com.cloudera.oryx.app.serving.als.MostPopularItemsTest#testMostPopularCSV], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=3}
status: NOPATCH
dateEnd: Wed Apr 19 00:51:56 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@3d8c8ca2', projectTests=[com.cloudera.oryx.app.serving.als.MostPopularItemsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #30
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.CompressedResponseTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.CompressedResponseTest#testGzipResponse, com.cloudera.oryx.app.serving.als.CompressedResponseTest#testDeflateResponse], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:04 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@3f1f6171', projectTests=[com.cloudera.oryx.app.serving.als.CompressedResponseTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #31
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.FeatureImportanceTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.rdf.FeatureImportanceTest#testImportance, com.cloudera.oryx.app.serving.rdf.FeatureImportanceTest#testAllImportance], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:14 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@7eb07a68', projectTests=[com.cloudera.oryx.app.serving.rdf.FeatureImportanceTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #32
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.AllItemIDsTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.AllItemIDsTest#testAllItemIDs, com.cloudera.oryx.app.serving.als.AllItemIDsTest#testAllItemIDsCSV], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:24 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@59c550bd', projectTests=[com.cloudera.oryx.app.serving.als.AllItemIDsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #33
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.AllUserIDsTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.AllUserIDsTest#testAllItemIDs, com.cloudera.oryx.app.serving.als.AllUserIDsTest#testAllUserIDsCSV], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:32 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@93710c6', projectTests=[com.cloudera.oryx.app.serving.als.AllUserIDsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #34
location: FailureLocation{className='com.cloudera.oryx.app.serving.als.PopularRepresentativeItemsTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.als.PopularRepresentativeItemsTest#testPopularRepresentativeItems, com.cloudera.oryx.app.serving.als.PopularRepresentativeItemsTest#testPopularRepresentativeItemsCSV], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:39 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@55b3da15', projectTests=[com.cloudera.oryx.app.serving.als.PopularRepresentativeItemsTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #35
location: FailureLocation{className='com.cloudera.oryx.app.serving.kmeans.DistanceToNearestTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.kmeans.DistanceToNearestTest#testDistance2, com.cloudera.oryx.app.serving.kmeans.DistanceToNearestTest#testDistance], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:44 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@5561f343', projectTests=[com.cloudera.oryx.app.serving.kmeans.DistanceToNearestTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #36
location: FailureLocation{className='com.cloudera.oryx.app.serving.rdf.ClassificationDistributionTest', failingMethods=[], erroringMethods=[com.cloudera.oryx.app.serving.rdf.ClassificationDistributionTest#testDistribution2, com.cloudera.oryx.app.serving.rdf.ClassificationDistributionTest#testDistribution], failures=[FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}, FailureType{failureName='javax.ws.rs.InternalServerErrorException', failureDetail='HTTP 500 Internal Error', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Wed Apr 19 00:52:48 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/OryxProject/oryx/223328603/app/oryx-app-serving/src/main/java], projectClasspath='[Ljava.net.URL;@4fd7b798', projectTests=[com.cloudera.oryx.app.serving.rdf.ClassificationDistributionTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



